guides/v2.0/contributor-guide/contributing.md

```diff
@@ -214,10 +214,15 @@ Refer to the following table for a description of each label. These labels are a
       <tr>
          <td><img src="{{ site.baseurl }}common/images/github_Cannot_Reproduce.png" alt="Can Not Reproduce"/></td>
          <td>The Community Engineering Team can not reproduced the issue following steps to reproduce.</td>
-      </tr>  
+      </tr>
       <tr>
          <td><img src="{{ site.baseurl }}common/images/github_non-issue.png" alt="Non Issue"/></td>
          <td>A described behavior in the issue description is valid and shouldn't be changed in Magento code base.</td>
       </tr>
    </tbody>
+
+<h2 id="triage">Help triage issues <a href="https://www.codetriage.com/magento/magento2"><img src="https://www.codetriage.com/magento/magento2/badges/users.svg" /></a></h2>
```

Comment:
Please add the `target="_blank"` attribute to the link to open in a new browser tab.